packages/fast-components-react-msft/src/index.ts

```diff
@@ -1,3 +1,4 @@
 import Button from "./button";
+import Media from "./media";
 
-export { Button };
+export { Button, Media };
```

Comment:
I think we'll want to keep the exports coupled with their respective import for mainainability